workspaces/frontend/vite.config.ts

```diff
@@ -2,6 +2,10 @@ import { defineConfig } from "vite";
 import solidPlugin from "vite-plugin-solid";
 
 export default defineConfig({
+    define: {
+        // TODO: Remove duplication with backend/src/settings
```

Comment:
todo comment

Reply:
That's out of scope of this PR, it will be addressed when we introduce the shared library.